reportToRead.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
```

Comment:
It's more common to store additional files in 'resources' package

Reply:
Could you please explain in more detail what exactly needs to be done? For all comments, I don't understand.

Reply:
The folder structure of a Java project is crucial for organizing the code, managing dependencies, and ensuring that the project is easily maintainable. In a typical folder structure for a standard Java project 'resources' folder stores non-Java files needed by the application, such as configuration files (application.properties, logback.xml), static assets (images, HTML files), or localization files.
https://www.oreilly.com/library/view/java-ee-8/9781788833776/43a61672-4663-4de6-b336-a36a3c5a045e.xhtml